tests/FakeItEasy.Tests.Approval/ApiApproval.cs

```diff
@@ -17,42 +17,73 @@ public class ApiApproval
         [MethodImpl(MethodImplOptions.NoInlining)]
         public void ApproveApi45()
         {
-            ApproveApi("net45");
+            ApproveApi("FakeItEasy", "net45");
         }
 
         [Fact]
         [UseReporter(typeof(DiffReporter))]
         [MethodImpl(MethodImplOptions.NoInlining)]
         public void ApproveApi40()
         {
-            ApproveApi("net40");
+            ApproveApi("FakeItEasy", "net40");
         }
 
         [Fact]
         [UseReporter(typeof(DiffReporter))]
         [MethodImpl(MethodImplOptions.NoInlining)]
         public void ApproveApiNetStd16()
         {
-            ApproveApi("netstandard1.6");
+            ApproveApi("FakeItEasy", "netstandard1.6");
         }
 
         [Fact]
         [UseReporter(typeof(DiffReporter))]
         [MethodImpl(MethodImplOptions.NoInlining)]
         public void ApproveApiNetStd20()
         {
-            ApproveApi("netstandard2.0");
+            ApproveApi("FakeItEasy", "netstandard2.0");
         }
 
+        [Fact]
+        [UseReporter(typeof(DiffReporter))]
+        [MethodImpl(MethodImplOptions.NoInlining)]
+        public void ApproveApiNetStd21()
+        {
+            ApproveApi("FakeItEasy", "netstandard2.1");
+        }
+
+        [Fact]
+        [UseReporter(typeof(DiffReporter))]
         [MethodImpl(MethodImplOptions.NoInlining)]
-        private static void ApproveApi(string frameworkVersion)
+        public void ApproveExtensionsValueTaskApi45()
+        {
+            ApproveApi("FakeItEasy.Extensions.ValueTask", "net45");
+        }
+
+        [Fact]
+        [UseReporter(typeof(DiffReporter))]
+        [MethodImpl(MethodImplOptions.NoInlining)]
+        public void ApproveExtensionsValueTaskApiNetStd16()
+        {
+            ApproveApi("FakeItEasy.Extensions.ValueTask", "netstandard1.6");
+        }
+
+        [Fact]
+        [UseReporter(typeof(DiffReporter))]
+        [MethodImpl(MethodImplOptions.NoInlining)]
+        public void ApproveExtensionsValueTaskApiNetStd20()
+        {
+            ApproveApi("FakeItEasy.Extensions.ValueTask", "netstandard2.0");
+        }
+
+        private static void ApproveApi(string projectName, string frameworkVersion)
         {
             string codeBase = Assembly.GetExecutingAssembly().CodeBase;
             UriBuilder uri = new UriBuilder(new Uri(codeBase));
             string assemblyPath = Uri.UnescapeDataString(uri.Path);
             var containingDirectory = Path.GetDirectoryName(assemblyPath);
             var configurationName = new DirectoryInfo(containingDirectory).Parent.Name;
-            var assemblyFile = $@"..\..\..\..\..\src\FakeItEasy\bin\{configurationName}\{frameworkVersion}\FakeItEasy.dll";
+            var assemblyFile = $@"..\..\..\..\..\src\{projectName}\bin\{configurationName}\{frameworkVersion}\{projectName}.dll";

```

Comment:
Note that I removed the `NoInlining` attribute from this method, because it wasn't necessary. It only matters on the test method itself, which ApprovalTests uses to infer the name of the approved/received files.

(and this method would never have been anyway, it's too long)